### PR TITLE
CE Locker Vengeance: The Removaling + Welding locker consistency changes + Empty lockers for mappers

### PR DESCRIPTION
**Pull request by KathrinBailey**
Removes junk from the CE locker.
Adds empty engineering lockers.
I don't know how to do this as a child and not have populate contents fill the child.

## About The Pull Request

Removes these items from the CE's locker:
• Airlock Painter
• Multitool
• Gas Mask
• Hazard Vest
• Brown Sneakers
• Mechanical Toolbox
Adds an empty electrical tools closet for mappers.
Adds an empty engineering closet for mappers.
Changes the welding tools in the welding tool closet from normal to industrial welding tools as the engineering vendors have normal ones, normal ones can be found everywhere, and engineers start with industrial ones.

## Why It's Good For The Game

Open the locker and close it, and things spill out, this should no longer happen. Adds consistency with the welding tools. Gives a little bit more freedom to mappers.

## Changelog
:cl:
add: Empty engineering lockers for mappers.
add: Industrial welding tools to the engineer welding locker.
del: Removed the multitool, airlock painter, mechanical toolbox, brown sneakers, hazard vest and airlock painter from the CE's locker.
/:cl:

### CODE REVIEW

**Comment by dapnee** [comment]
How dare you steal the CE's extra pair of fancy brown shoes.

**Comment by KathrinBailey** [comment]
The brown shoes are the only thing I was considering not removing from that list since its a head of staff thing, but they spawn with the shoes. I'd put it back but eehh the only footwear the CE should wear is either the magboots or some cool snowflake special shoe if you don't feel like taking an energy sword up the ass in maint for your shoes because traitor objective.
Regardless, if you change shoes as a CE, you still have access to the brown shoes.

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 1]
```diff
@@ -56,6 +50,12 @@
 	for(var/i in 1 to 3)
 		new /obj/item/multitool(src)
 
+/obj/structure/closet/secure_closet/engineering_electrical_empty
```
Comment (Ghommie) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: Make these subtypes of the real things, but with PopulateContents() returning nothing.

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 2]
```diff
@@ -84,6 +84,10 @@
 	new /obj/item/clothing/glasses/meson/engine(src)
 	new /obj/item/storage/box/emptysandbags(src)
 
+/obj/structure/closet/secure_closet/engineering_personal_empty
```
Comment (Ghommie) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: Same here.

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 3]
```diff
@@ -12,18 +12,12 @@
 	new /obj/item/clothing/head/hardhat/white(src)
 	new /obj/item/clothing/head/hardhat/weldhat/white(src)
 	new /obj/item/clothing/gloves/color/yellow(src)
-	new /obj/item/clothing/shoes/sneakers/brown(src)
```
Comment (Ghommie) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: The "backup" pair of shoes can stay.

**Review by Ghommie — APPROVED** [review]

**Review by Ghommie — CHANGES_REQUESTED** [review]
Whoops. See above.

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm (hunk 1 above)
Comment (KathrinBailey) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: Once I find out how to make PopulateContents() not affect the subtype I will, so this PR is going to be open for a few days.

Comment (Ghommie) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]:
```
/obj/structure/closet/secure_closet/engineering_electrical/empty/PopulateContents()
	return
```

Is it too hard, you little dum dum?

**Review by Ghommie — APPROVED** [review]

**Pull request merged**